### DISCUSSION TITLE
Is sparse set a special BiMap?

### DISCUSSION THREAD

**Discussion by fililili** [skypjack/entt#944] [General]
Seems that a sparse set is just a special BiMap<Entity, std::size_t>, is that true?

**Comment by skypjack** [comment]
Not really.
`EnTT` contains a highly customized and optimized implementation. However, you can find the basic details [here](https://programmingpraxis.com/2012/03/09/sparse-sets/).
This should clear your doubts. Let me know if this isn't the case though.
